### DISCUSSION TITLE
Google Auth Popup - No Dark Background Overlay?

### DISCUSSION THREAD

**Discussion by tommyc38** [capawesome-team/capacitor-firebase#449] [Q&A] [answered]
See the image below.  Everything works great but the popup doesn't have a dark background making it look like an overlay.  Is there somewhere I can change this to make it look better?

<img width="339" alt="Screenshot 2023-10-07 at 1 55 03 PM" src="https://github.com/capawesome-team/capacitor-firebase/assets/10190567/30cfb639-ab67-4b82-b2a5-291e232343fd">

<img width="322" alt="Screenshot 2023-10-07 at 1 54 47 PM" src="https://github.com/capawesome-team/capacitor-firebase/assets/10190567/5fdcd84e-0203-429b-8b53-e33fbcf19b65">

**Comment by robingenz** [comment, marked as answer]
Thank you for your request.
Your app UI is completely up to you.
For example, you could show an ion-backdrop overlay before calling the plugins method, see https://ionicframework.com/docs/api/backdrop

**Reply by tommyc38** [reply]
Thank you for the suggestion.  Works great!
